transforms/create-element-to-jsx.js

```diff
@@ -60,11 +60,20 @@ module.exports = function(file, api, options) {
     }
   };
 
+  const jsxIdentifierNameFor = node => {
```

Comment:
Can we change this to just return the node itself instead of generating a string?